OpenTok/OpenTok.cs

```diff
@@ -534,6 +538,10 @@ public Broadcast StartBroadcast(string sessionId, Boolean hls = true, List<Rtmp>
                 {
                     throw new OpenTokArgumentException("Could not set the layout. Either an invalid JSON or an invalid layout options.");
                 }
+                else if (layout.ScreenShareType != null && layout.Type != BroadcastLayout.LayoutType.BestFit)
+                {
+                    throw new OpenTokArgumentException($"Could not set screenShareLayout, when screenShareType is set, layout.Type must be bestFit, was {layout.ScreenShareType}");
```

Comment:
Same comma fix as the other
```suggestion
                    throw new OpenTokArgumentException($"Could not set screenShareLayout, when screenShareType is set layout.Type must be bestFit, was {layout.ScreenShareType}");
```